boards/DIM/Src/main.c

```diff
@@ -645,6 +649,7 @@ void RunTaskCanRx(void const *argument)
         Io_SharedCan_DequeueCanRxMessage(&message);
         Io_CanRx_UpdateRxTableWithMessage(
             App_DimWorld_GetCanRx(world), &message);
+        Io_ErrorTable_Update(App_DimWorld_GetErrorTable(world), &message);
```

Comment:
Wasn't sure if `Io_ErrorTable_Update` should be called inside `Io_CanRx_UpdateRxTableWithMessage`. Left them separate for now.